### ISSUE TITLE
ngbButton Accessibility issue from input having opacity of zero

### ISSUE THREAD

**Issue by croeber** [ng-bootstrap/ng-bootstrap#4112]
### Bug description:
ngbButton directives fail accessibility because the <input> has an opacity of 0.
It is being reported in the ARC Tool ( A Chrome extension )
https://chrome.google.com/webstore/detail/arc-toolkit/chdkkkccnlfncngelccgbgfmjebmkmce?hl=en

### **Here is the ARC Report**
```
<input _ngcontent-ckn-c270="" type="checkbox" ngbbutton="" class="ng-untouched ng-pristine ng-valid">
contrast: 4.26:1
insufficientNormalTextContrast
Error: insufficient color contrast between foreground text and the background
Recommend: ensure text has a minimum contrast ratio of 4.5:1
```

### Link to minimally-working StackBlitz that reproduces the issue:
https://angular-cxmezt.stackblitz.io
https://stackblitz.com/edit/angular-cxmezt?file=src/app/buttons-checkbox.html

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: Angular 10.2.5

ng-bootstrap: 8.0.4

Bootstrap: 4.6.0

**Comment by maxokorokov** [comment]
Buttons were removed in v13 → https://ng-bootstrap.github.io/releases/12.x/#/components/buttons/overview
